src/Stepper/Step.less

```diff
@@ -0,0 +1,171 @@
+@import "../less/fonts.less";
+@import (reference) "../less/index";
+
+@stepIconWidth: 1.75rem; // 28px
+@iconFocusShadowThickness: @size_2xs; // 4px
+@titleFontSize: 1.25rem; // 20px
+@alertSize: 0.375rem; // 6px
+
+.Step {
+  .flexbox();
+  .alignItems(flex-start);
+  .padding--y--2xs;
+  .text--medium;
+  .text--regular;
+  .padding--none;
+  text-align: left;
+  .text--line-height-3;
+}
+
+button.Step {
+  .button--reset();
```

Comment:
this will override any intersecting styles in `.Step` above - i believe one way to solve this and make sure the button and non-button styling stay in sync is to add a reference to `.Step` as a mixin:

```less
button.Step {
  .button--reset();
  .Step();
}
```